app/src/main/res/layout/feeditem_fragment.xml

```diff
@@ -171,6 +171,20 @@
             android:layout_height="1dp"
             android:background="?android:attr/dividerVertical"/>
 
+        <TextView
+            android:id="@+id/noMediaMsg"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:visibility="visible"
```

Comment:
I think it would be better to default to "gone" - AntennaPod is a podcast app, after all, so that prevents screen flickering for most users.

Reply:
whoops, I swapped it to visible when I was looking at the layout in AS, forgot to change it back.